tests/data/modules/tests/model/_init.cf

```diff
@@ -0,0 +1,7 @@
+entity Test:
+end
+
+implementation test for Test:
+end
```

Comment:
Why even have an implementation if it's a NOOP?

Reply:
No reason, I copy/pasted it from an existing test above and didn't remove it.